setup.py

```diff
@@ -15,13 +15,22 @@
 # this program. If not, see <http://www.gnu.org/licenses/>.
 #
 
-from distutils.core import setup
+from setuptools import setup, find_packages
 
-setup(name='asynk',
+setup(
+      name='asynk',
       version='0.4.1',
       description='Synchronizes Google Contacts, BBDB and Outlook',
       author='Sriram Karra',
       author_email='karra.etc@gmail.com',
       url='https://karra-asynk.appspot.com/',
-      packages=['asynk'],
-     )
+      packages=['asynk','lib'],
+      install_requires=['demjson','vobject','httplib2'],
+#     install_requires=['iso8601','demjson','vobject','httplib2'],#,'caldavclientlibrary'],#,'apiclient'],#,'atom','distribute','vobject'],
```

Comment:
First, thanks for these changes. I am happy to incorporate the changes because at a high level I am OK with the idea of making the package installable, which will make it more distributable. However, I will have to admit my ignorance in how distutils works. So please bear with me here...

When I apply your entire patch to a working directory and run ```python setup.py install``` it creates asynk/ and lib/ in ```site-packages```. However the submodules are nowhere to be found. The submodules are critical to get a bunch of datastores working, like CardDAV etc.

Questions:

- What datastores did you test?
- In particular did you try Google, CardDAV , MS Exchange?
- In general, how does distutils deal with submodules? We really need to find a wya to get the right versions of the submodules even when there are public versions available because there are some patches to the version I am bundling here.